xarray/tutorial.py

```diff
@@ -77,9 +82,26 @@ def load_dataset(name, cache=True, cache_dir=_default_cache_dir,
             """
             raise IOError(msg)
 
-    ds = _open_dataset(localfile, **kws).load()
+    ds = _open_dataset(localfile, **kws)
 
     if not cache:
         _os.remove(localfile)
```

Comment:
We definitely need to load the dataset first if not caching -- otherwise you'll get errors when trying to use it.